evm/src/cpu/contextops.rs

```diff
@@ -83,9 +84,13 @@ fn eval_packed_get<P: PackedField>(
     // If the opcode is GET_CONTEXT, then lv.opcode_bits[0] = 0.
     let filter = lv.op.context_op * (P::ONES - lv.opcode_bits[0]);
     let new_stack_top = nv.mem_channels[0].value;
-    yield_constr.constraint(filter * (new_stack_top[0] - lv.context));
-    for &limb in &new_stack_top[1..] {
-        yield_constr.constraint(filter * limb);
+    // Context is scaled by 2^64, hence stored in the 3rd limb.
+    yield_constr.constraint(filter * (new_stack_top[2] - lv.context));
+
+    for (i, &limb) in new_stack_top.iter().enumerate() {
```

Comment:
There are always three limbs here yes? If so, why bother iterating? Just `yield_constr.constraint()` twice up front with explicit index `0` and `1`?

Reply:
There are 8 limbs per memory channel (i.e. holding a `U256` split into 8 `u32` limbs).

Reply:
Ok, fair enough then (I guess you could have used `filter()` to exclude index `2` but this is fine too.

Reply:
filter would be cleaner though, will update it!